app/models/user.rb

```diff
@@ -17,6 +17,12 @@ class User < ApplicationRecord
   has_many :project_members, dependent: :destroy
   has_many :projects, through: :project_members
 
+  scope :alphabetical, -> { order(:name) }
+
+  def available_projects
+    admin ? Project.all : projects
```

Comment:
Similar to the other comment on User - could this only send the relevant project attrs? I believe from the Vue component they are `name` and `updated_at`

Reply:
I just made a change that uses `.select(:id, :name, :updated_at)` in `Projects#index` since there is a chance that we may want to use this method for other purposes in the future